packages/contracts-bedrock/scripts/DeployOPChain.s.sol

```diff
@@ -70,6 +71,17 @@ contract DeployOPChainInput is BaseDeployIO {
         }
     }
 
+    function set(bytes4 _sel, AnchorStateRegistry.StartingAnchorRoot[] memory _roots) public {
```

Comment:
If we're only deploying straight to permissioned games for now (which I believe we are, cause of the cyclic dependency), does that help us cut any scope by not needing to add support and tests for setting initial anchor roots?

Reply:
I don't think the lift is huge, do you see a much missing in the way of testing here? 

Before cutting it, we'd need to discuss with Proofs. My impression is that empty anchor roots is a possibility but not necessarily the default with the permissioned game.

Reply:
Ok, just checked where `startingAnchorRoots` is used and you're right the current testing is sufficient. Just wanted to raise this as an option but it seems unnecessary to cut this scope, so I'll resolve this